manual_run/map_columns.py

```diff
@@ -0,0 +1,53 @@
+from typing import Dict, List
+
+from manual_run.variables import Variables
+
+
+class MapColumns:
+    @staticmethod
+    def process_row(row: List[str], header_row: List[str]) -> Dict:
+        """Convert the non-integer values in the CSV file to integer values
+
+        Args:
+            row (List[str]): current row to be processed into teh correct integer values
+            header_row (List[str]): the header row of the CSV file
+
+        Returns:
+            Dict: a dictionary containing the list of integer values of the
+            combined attribute_options and attribute_handlers maps in variables.py
+        """
+        attribute_options_int: Dict[str, List[int]] = {}
+        for i, head in enumerate(header_row):
+            int_value = -1
+            if head in Variables.attribute_options:
+                if row[i] in Variables.attribute_options[head]:
+                    int_value = Variables.attribute_options[head].index(row[i])
+                attribute_options_int[head] = int_value
+            elif head in Variables.attribute_handlers:
+                int_value = Variables.attribute_handlers[head](row[i])
+                attribute_options_int[head] = int_value
+        return attribute_options_int
+
+    @staticmethod
+    def revert_timeslot(time_slot: int) -> str:
```

Comment:
Why do we need these revert methods again?

Reply:
The only way I can think of reverting these integer values back to their string value would be by calling a function to do so. I over looked how simple this would be to break down into one function. Is doing this in a function like the following ok?
```
def revert_value(attribute_key: str, value: int) -> str:
    """Revert integer value to the original string based on the attribute key."""
    if attribute_key in Variables.attribute_options:
        options = Variables.attribute_options[attribute_key]
        if value < len(options):
            return options[value]
    return ""
```